src/Symfony/Component/Messenger/Transport/Sender/SendersLocatorInterface.php

```diff
@@ -9,19 +9,20 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Messenger\Transport\Sender\Locator;
+namespace Symfony\Component\Messenger\Transport\Sender;
 
-use Symfony\Component\Messenger\Envelope;
 use Symfony\Component\Messenger\Transport\Sender\SenderInterface;
 
 /**
  * @author Samuel Roze <samuel.roze@gmail.com>
  * @author Tobias Schultze <http://tobion.de>
  */
-interface SenderLocatorInterface
+interface SendersLocatorInterface
 {
     /**
-     * Gets the sender (if applicable) for the given message object.
+     * Gets the senders for the given message.
+     *
+     * @return iterable|SenderInterface[]
      */
-    public function getSender(Envelope $envelope): ?SenderInterface;
+    public function getSenders(string $name, ?bool &$handle = false): iterable;
```

Comment:
It's not documented what these params mean.

Reply:
sorry didn't answer: in order to ease using the method with uninitialized variables.